src/components/popover/popover.jsx

```diff
@@ -86,14 +104,18 @@ const Popover = forwardRef(function Popover(props, ref) {
 });
 
 Popover.propTypes = {
+  autoflush: PropTypes.bool,
```

Comment:
We need tests for this prop.

Reply:
Mentioned on coscreen but for posterity:

I ran into struggling how to test this because `getBoundingClientRect` always returns 0's (`{x: 0, y: 0, left: 0, ... }`) in jsdom. Some people have gotten around this by overwriting `Object.prototype.getBoundingClientRect` but that felt super wrong to do. 

I'm going to take a stab at mocking it out on the individual section ref. Another idea is to extract this as a hook and test that hook instead, but that doesn't really guarantee that this component works and it's also not a super reusable hook to begin with.

Reply:
98c0bdc2

Looks like stubbing the element's function worked, just takes a little finagling.